### PR TITLE
add doc around fusing multiple loras.

### PR DESCRIPTION
**Pull request by sayakpaul**
# What does this PR do?

Adds documentation for fusing multiple LoRAs. Presenting cyborg pikachu (thanks to @apolinario):

image

### CODE REVIEW

File: docs/source/en/training/lora.md [hunk 1]
```diff
@@ -378,6 +378,56 @@ images_fusion = pipe(
...
+
+Let's see them in action:
+
+```python
+prompt = "cyborg style pikachu"
+image = pipe(prompt, num_inference_steps=30, guidance_scale=7.5).images[0]
+```
+
+![cyborg_pikachu](https://huggingface.co/datasets/diffusers/docs-images/resolve/main/cyborg_pikachu.png)
+
```
Comment (apolinario) [inline docs/source/en/training/lora.md:430]:
```suggestion
<Tip warning={true}>

Currently,unfusing multiple LoRA checkpoints is not possible. If multiple LoRAs are fused into the `unet`, the `pipe.unfuse_lora()` method doesn't work. 

</Tip>
```

File: docs/source/en/training/lora.md [hunk 2]
```diff
@@ -378,6 +378,56 @@ images_fusion = pipe(
 ).images
 ```
 
+## Working with multiple LoRA checkpoints
+
+<Tip warning={true}>
+
+Our support for loading multiple LoRA checkpoints is currently limited and there might be rough edges involved.
```
Comment (apolinario) [inline docs/source/en/training/lora.md:385]: I would remove it here and move it below to the suggested part

Comment (apolinario) [reply docs/source/en/training/lora.md:385]: I think "limited support" and "rough edges" without specification may be a bit discouraging. I think it's best to maybe mention the known limitations in a more specific way.

File: docs/source/en/training/lora.md [hunk 3]
```diff
@@ -378,6 +378,56 @@ images_fusion = pipe(
...
+Then let's two LoRA checkpoints and fuse them with specific `lora_scale` values:
+
+```python
+# LoRA one.
+pipe.load_lora_weights("goofyai/cyborg_style_xl")
+pipe.fuse_lora(lora_scale=0.5)
+
+# LoRA two.
+pipe.load_lora_weights("TheLastBen/Pikachu_SDXL")
+pipe.fuse_lora(lora_scale=0.5)
```
Comment (apolinario) [inline docs/source/en/training/lora.md:413]:
```suggestion
pipe.fuse_lora(lora_scale=0.7)
```

Comment (apolinario) [reply docs/source/en/training/lora.md:413]: found this to be a better default

File: docs/source/en/training/lora.md [hunk 4]
```diff
@@ -378,6 +378,56 @@ images_fusion = pipe(
...
+)
+pipe.to("cuda")
+```
+
+Then let's two LoRA checkpoints and fuse them with specific `lora_scale` values:
+
+```python
+# LoRA one.
+pipe.load_lora_weights("goofyai/cyborg_style_xl")
+pipe.fuse_lora(lora_scale=0.5)
```
Comment (apolinario) [inline docs/source/en/training/lora.md:409]:
```suggestion
pipe.fuse_lora(lora_scale=0.7)
```

Comment (apolinario) [reply docs/source/en/training/lora.md:409]: found this to be a better default

**Review by patrickvonplaten — APPROVED** [review]

**Pull request merged**